napari/_app_model/actions/_view_actions.py

```diff
@@ -0,0 +1,45 @@
+from typing import TYPE_CHECKING, List
```

Comment:
Nit: it might be nice to add a summary doc string here as in `_layer_actions.py`: 
https://github.com/napari/napari/blob/9ea0159ad2b690556fe56ce480886d8f0b79ffae/napari/_app_model/actions/_layer_actions.py#L1-L9

Also this may be a naive question, why is this file in `actions/`required for the 'view' menu but not for the 'file' menu in #4865?

Reply:
because all of the actions in our file menu currently require Qt (i.e., they raise a dialog of something), and therefore live in the _qt folder at `_qt/_qapp_model/qactions/_file.py`